developer/src/kmc/src/commands/buildClasses/BuildProject.ts

```diff
@@ -41,6 +41,11 @@ class ProjectBuilder {
       return false;
     }
 
+    // Give a hint if the project is v1.0
+    if(this.project.options.version != '2.0') {
```

Comment:
Wouldn't a version comparison (< 2.0) future-proof this check?

Reply:
Perhaps, but we'll need to revisit all this anyway if we want to upgrade project file versions again, and we may want to do something different for that case.